src/DnDCombatTracker.Core/Character.cs

```diff
@@ -50,6 +52,7 @@ public Character(string name, short initiative, short? maxHP, short? HP, short?
             DeathSaves_Fail = fail;
             DeathSaves_Success = success;
             Conditions = conditions;
+            Note = notes;
```

Comment:
Perhaps you could rename the property `Note` to `Notes`, it's a better name. I see your constructor param name is already notes and that fits better.

perhaps you also have to update the UI to reflect this, idk if we are showing 'Note' there right now

Reply:
Sounds good, I can make the changes tonight

Reply:
Nevermind, I resolved it.

Reply:
![afbeelding](https://user-images.githubusercontent.com/7312681/66254736-f7260080-e77a-11e9-9dc3-2c94a56290b6.png)


Seems i found a bug.

I see there are now 2 note fields, one in details, one in the "information" tab.

When I have multiple characters and I select a different one, the information tab updates with the information of the selected character.

The notes do not change, although they should.

Reply:
I see that, I'll be looking into it.  I think I missed the information panel update

Reply:
Resolved with the latest commit